### ISSUE TITLE
how to split big dataset into training/valid/test se?

### ISSUE THREAD

**Issue by Just4Uzi** [brightmart/ai_law#1]
请问在large数据集上是怎么划分train/test/valid的呢

**Comment by brightmart** [comment]
Hi,


there is a valid/test set in small dataset. i just replace training data from large dataset. valid/test is same as small dataset.


bright


________________________________
发件人: SlashChven <notifications@github.com>
发送时间: 2018年6月19日 16:58
收件人: brightmart/ai_law
抄送: Subscribed
主题: [brightmart/ai_law] 关于数据集划分问题 (#1)


请问在large数据集上是怎么划分train/test/valid的呢

―

**Comment by Just4Uzi** [comment]
Hi,


I found the large dataset contain small valid/test dataset.  Is this data division method not suitable?

Slash

**Comment by brightmart** [comment]
thanks for your informaiton. i think that's why the local performance is a little lower than offlne performance.


you can just split train/valid/test from large dataset. for example given 10k to valid, 10k to test, the rest for training.


but in fact, even my setting, it is still works.


________________________________
发件人: SlashChven <notifications@github.com>
发送时间: 2018年6月19日 19:27
收件人: brightmart/ai_law
抄送: brightmart; Comment
主题: Re: [brightmart/ai_law] 关于数据集划分问题 (#1)


Hi,

I found the large dataset contain small valid/test dataset. Is this data division method not suitable?

Slash

―
